mtx/sidecar/package.json

```diff
@@ -30,8 +14,11 @@
   "engines": {
     "node": "^16"
   },
+  "cds": {
+    "profile": "mtx-sidecar"
+  },
   "scripts": {
     "start": "cds-serve",
-    "build": "npm install && npx cds build ../.. --for mtx-sidecar && cd gen && npm install"
+    "build": "cds build ../.. --for mtx-sidecar --production && cd gen && npm install"
```

Comment:
Productive applications should go for the package-lock approach in order to ensure reproducible deployments